app/commands/v2/publish.rb

```diff
@@ -177,6 +177,7 @@ def clear_published_item_of_different_locale_but_matching_base_path
         # This enables changing the locale of some content where a
         # published edition for the previous locale still exists.
         published_edition_for_different_locale.substitute
+        SubstitutionHelper.substitute_message(published_edition_for_different_locale)
```

Comment:
Could the first line of the commit message be shortened here, so it fits onto one line?

Reply:
Done